LICENSE

```diff
@@ -1,8 +1,6 @@
-All parts, except the contents of the documentation module, are licenced
-under Apache License v2.0. See the license text below.
-
-The documentation is licensed under Creative Commons CC-BY-ND 2.0
-(http://creativecommons.org/licenses/by-nd/2.0/legalcode).
+This project dual-licensed under Apache 2.0, for free components, and
```

Comment:
Having 2 licenses in the file is confusing, I would just write a sentence with : please visit the license file in each module, or the same text than in the readme file

Reply:
done